tests/test_e2e_12_mef_eline.py

```diff
@@ -133,7 +133,7 @@ def test_create_schedule_by_date(self, disabled_circuit_id):
             after one minute. """
 
         # Schedule by date to next minute
-        ts = datetime.now() + timedelta(seconds=60)
+        ts = datetime.utcnow() + timedelta(seconds=60)
```

Comment:
The tests related to these are passing, the previous bug was mostly caused by the formatting `ts.strftime("%Y-%m-%dT%H:%M:%S.000Z")`. Regardless of the timezone that "Z" at the end would automatically turn it into UTC. So maybe that was the problem since the others differ in formatting and instead utilize `"%Y-%m-%dT%H:%M:%S+0000"
`.

Reply:
Indeed. Good catch. Let's close this thread.